core/templates/dev/head/css/oppia.css

```diff
@@ -53,8 +53,9 @@ md-input-group.md-default-theme label
   margin-bottom: 0;
 }
 
-.md-button-success {
-  color: #009688;
+.oppia-learner-continue-button {
```

Comment:
Use `.md-button.oppia-learner-continue-button` instead. The reason you're getting a different colour than expected is due to the CSS specificity rules, which you can read about [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity). The extra class fixes this.

By the way, you can see what CSS styles are being applied to an element using the browser console: https://developer.chrome.com/devtools/docs/elements-styles . This allows for easy in-browser experimentation.